spring-kafka-test/src/main/java/org/springframework/kafka/test/rule/KafkaEmbedded.java

```diff
@@ -222,6 +240,31 @@ public void before() throws Exception { //NOSONAR
 		System.setProperty(SPRING_EMBEDDED_ZOOKEEPER_CONNECT, getZookeeperConnectionString());
 	}
 
+	public Properties createBrokerProperties(int i) {
+		if (clientVersion.startsWith("1.0.")) {
```

Comment:
And here I think would be enough to check `testUtilsCreateBrokerConfigMethod == null` instead - should be much efficient than String manipulations.